plio/tests.py

```diff
@@ -993,6 +993,136 @@ def test_question_metrics_answers_provided(self):
         self.assertEqual(response.data["accuracy"], 58.33)
         self.assertEqual(response.data["has_survey_question"], False)
 
+    def test_question_mode_returned_if_no_sessions(self):
+        # seed items
+        item_1 = Item.objects.create(type="question", plio=self.plio_1, time=1)
+
+        # seed questions
+        Question.objects.create(
+            type="mcq",
+            item=item_1,
+            text="test",
+            options=["", ""],
+            correct_answer=0,
+            survey=True,
+        )
+
+        response = self.client.get(
+            f"/api/v1/plios/{self.plio_1.uuid}/metrics/",
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data, {"has_survey_question": True})
+
+    def test_question_metrics_answers_provided_if_survey_question_exists(self):
```

Comment:
this is what you intended right? `test_question_metrics_calculated_using_non_survey_questions`
not clear from the current name - had to read the code to figure it out.